src/main/java/com/polishuchenko/bookstore/service/cartitem/impl/CartItemServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.polishuchenko.bookstore.service.cartitem.impl;
+
+import com.polishuchenko.bookstore.dto.cartitem.CartItemResponseDto;
+import com.polishuchenko.bookstore.dto.cartitem.UpdateCartItemDto;
+import com.polishuchenko.bookstore.exception.EntityNotFoundException;
+import com.polishuchenko.bookstore.mapper.CartItemMapper;
+import com.polishuchenko.bookstore.model.CartItem;
+import com.polishuchenko.bookstore.repository.cartitem.CartItemRepository;
+import com.polishuchenko.bookstore.service.cartitem.CartItemService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CartItemServiceImpl implements CartItemService {
```

Comment:
I think it's redundant to create separate service. Put the methods to the same service where you put `addItem` method.